schema.sql

```diff
@@ -33,4 +33,26 @@ ALTER TABLE animals ADD FOREIGN KEY (species_id) REFERENCES species (id);
 
 ALTER TABLE animals ADD owner_id INTEGER;
 
-ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
\ No newline at end of file
+ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
+
+CREATE TABLE vets (
+    id INTEGER GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name varchar,
```

Comment:
- [ ] You should actually add a value after varchar(255). But as its a small thing, I am approving.